boards.txt

```diff
@@ -21268,6 +21265,24 @@ heltec_wireless_stick.menu.EraseFlash.none.upload.erase_cmd=
 heltec_wireless_stick.menu.EraseFlash.all=Enabled
 heltec_wireless_stick.menu.EraseFlash.all.upload.erase_cmd=-e
 
+heltec_wireless_stick.menu.Revision.Heltec_Wireless_Stick_V1=Heltec Wireless Stick V1 (SX1276)
+heltec_wireless_stick.menu.Revision.Heltec_Wireless_Stick_V1.build.board=HELTEC_WIRELESS_STICK_V1
+heltec_wireless_stick.menu.Revision.Heltec_Wireless_Stick_V1.build.variant=heltec_wireless_stick_V1
+heltec_wireless_stick.menu.Revision.Heltec_Wireless_Stick_V1.build.flash_size=4MB
+heltec_wireless_stick.menu.Revision.Heltec_Wireless_Stick_V1.build.partitions=default
+
+heltec_wireless_stick.menu.Revision.Heltec_Wireless_Stick_V2=Heltec Wireless Stick V2 (SX1276)
+heltec_wireless_stick.menu.Revision.Heltec_Wireless_Stick_V2.build.board=HELTEC_WIRELESS_STICK_V2
+heltec_wireless_stick.menu.Revision.Heltec_Wireless_Stick_V2.build.variant=heltec_wireless_stick
```

Comment:
@matthias-bs 
Variant `heltec_wereless_stick` does not exist as the only one existing is `heltec_wereless_stick_V1`.

If you want to specify variant for each version of the board, create `variants/heltec_wereless_stick_Vx/pins_arduino.h` for each version. That will be the best solution.

If the pins does not change over the versions, please rename the `heltec_wereless_stick_V1` just to `heltec_wereless_stick` and add it as default for the board and don't set it for each variant.
Thanks

Reply:
@P-R-O-C-H-Y
Now I'm confused. https://github.com/espressif/arduino-esp32/tree/master/variants/heltec_wireless_stick already exists and fits V2 and V3. I added https://github.com/espressif/arduino-esp32/tree/master/variants/heltec_wireless_stick_V1, because two pins (DIO1 and DIO2) are different in that version.

Reply:
Based on https://docs.heltec.cn/en/node/esp32/hardware_update_log.html

Reply:
Sorry @matthias-bs my bad. I thought you renamed that variant, but you have added the variant for the V1.

Reply:
Thanks!